SU2_CFD/src/fluid/CFluidModel.cpp

```diff
@@ -41,12 +41,19 @@
 #include "../../include/fluid/CPolynomialConductivityRANS.hpp"
 #include "../../include/fluid/CPolynomialViscosity.hpp"
 #include "../../include/fluid/CSutherland.hpp"
+#include "../../include/fluid/CCoolPropViscosity.hpp"
 #include "../../include/fluid/CConstantLewisDiffusivity.hpp"
+#include "../../include/fluid/CCoolPropConductivity.hpp"
+#include "../../include/fluid/CCoolPropConductivityRANS.hpp"
 
 unique_ptr<CViscosityModel> CFluidModel::MakeLaminarViscosityModel(const CConfig* config, unsigned short iSpecies) {
   switch (config->GetKind_ViscosityModel()) {
     case VISCOSITYMODEL::CONSTANT:
       return unique_ptr<CConstantViscosity>(new CConstantViscosity(config->GetMu_ConstantND(iSpecies)));
+    #ifdef USE_COOLPROP
```

Comment:
These #ifdefs should not spread throughout the code, if SU2 is not compiled with CoolProp these classes should be empty and throw an error in the constructor, but the logic resides only in "CoolProp-specific" files and not in other parts of the code.
```suggestion
```

Reply:
I deleted these #ifdefs in [CFluidModel.cpp].

Reply:
After I deleted these #ifdefs in [CFluidModel.cpp], the building process without CoolProp failed, as you can find from the github building error message.

I will try to modify CCoolPropViscosity.cpp and CCoolPropConductivity.cpp to solve this issue.

Reply:
@pcarruscag 

Hi, I solved this issue by moving #ifdefs to "CoolProp-specific" files: CCoolPropViscosity.hpp and CCoolPropConductivity.hpp. All checks on github passed.